sdk/python/lib/pulumi/runtime/rpc.py

```diff
@@ -814,7 +814,27 @@ def _unwrap_rpc_secret_struct_properties(value: Any) -> Tuple[Any, bool]:
             unwrapped, secret_element = _unwrap_rpc_secret_struct_properties(list_value)
             contains_secrets = contains_secrets or secret_element
             list_values.append(unwrapped)
-        return struct_pb2.ListValue(values=list_values), contains_secrets
+        return list_values, contains_secrets
+    # pylint: disable=no-member
+    if isinstance(value, struct_pb2.Value):
```

Comment:
disabled false negative lint error from pylint (mypy reports no issue) not sure why this is the case

Reply:
I *think* you can add this here to fix the issue without having to disable the linter: https://github.com/pulumi/pulumi/blob/master/sdk/python/.pylintrc#L187